packages/react/src/Accordion/Accordion.tsx

```diff
@@ -112,6 +113,11 @@ export type AccordionHeadingProps = HTMLAttributes<HTMLElement> & {
   toggleColor?: (typeof AccordionToggleColors)[number]
 }
 
+/**
```

Comment:
Let me know if you'd like this to be captured elsewhere

Reply:
👍 can you add this to a new tasklist on the same issue please, as remedial follow-up work.